<launch>

    <!--- Sim Time
    <param name="/use_sim_time" value="true" />
     -->

    <!--- Run Rviz-->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find lego_loam)/launch/test.rviz" />

    <!--- TF -->
    <node pkg="tf" type="static_transform_publisher" name="camera_init_to_map"  args="0 0 0 1.570795   0        1.570795 /map    /camera_init 10" />
    <node pkg="tf" type="static_transform_publisher" name="base_link_to_camera" args="0 0 0 -1.570795 -1.570795 0        /camera /base_link   10" />

    <!--- LeGO-LOAM
     <node pkg="lego_loam" type="pointPublish" name="pointPublish" output="screen" />
     -->
    <node pkg="lego_loam" type="imageProjection"    name="imageProjection"    output="screen"/>
    <node pkg="lego_loam" type="featureAssociation" name="featureAssociation" output="screen"/>
    <node pkg="lego_loam" type="mapOptmization"     name="mapOptmization"     output="screen"/>
    <node pkg="lego_loam" type="transformFusion"    name="transformFusion"    output="screen"/>


    <arg name="outputDirectory" default="/media/localization/新加卷/school/allSchool"  />
    <param  name="outputName" value="$(arg outputDirectory).txt" />


    <node pkg="rosbag" type="play" name="playe" output="screen" args="/media/localization/新加卷/kitti_RAW/kitti_2011_09_26_drive_0051_synced.bag">
        <remap from="/kitti/velo/pointcloud" to="/velodyne_points" />
    </node>

    <param name="PCD_directory" type="string" value="/media/localization/Localization/schoolData/2019024全校2/HDL/"/>

    <node pkg="loam_velodyne" type="getOdometry" name="getOdometry" output="screen">
    </node>
<!--
 <node name="recorder" pkg="rosbag" type="record" args="/laser_cloud_surround /integrated_to_init -O $(arg outputDirectory)" />
-->

</launch>
